concourse/steps/malware_scan.mako

```diff
@@ -65,35 +66,31 @@ filter_function = create_composite_filter_function(
   include_component_names=${filter_cfg.include_component_names()},
   exclude_component_names=${filter_cfg.exclude_component_names()},
 )
-
-image_references = [
-  resource.access.imageReference
-  for component, resource
-  in product.v2.enumerate_oci_resources(
-    component_descriptor=component_descriptor,
-  )
-  if filter_function(component, resource)
-]
-
+id = uuid.uuid4()
```

Comment:
using `id` as identifier is discouraged, I think, as it shadows the thus-named built-in